tsconfig.json

```diff
@@ -19,6 +19,7 @@
     "useUnknownInCatchVariables": false,
     "moduleResolution": "node",
     "resolveJsonModule": true,
+    "importHelpers": true,
```

Comment:
Just to clarify: this reduces code duplication, and nothing else? How is it related to browser support?

If this can be turned on without modifying browserslist, can you break it out into a separate very small PR, so we can get it in before resolving the final browserslist?